Jenkinsfile

```diff
@@ -9,8 +9,8 @@ pipeline {
     stage('main') {
       agent {
          dockerfile {
-            filename 'cufinufft/ci/docker/cuda10.1/Dockerfile-x86_64'
-            args '--gpus 1'
+            filename 'cufinufft/ci/docker/cuda11.0/Dockerfile-x86_64'
+            args '--gpus 2'
```

Comment:
Are you planning 2-GPU tests here?

Reply:
Yep. Since we have a test for this, I figured it'd be good to make sure it runs…